src/dmd/backend/cod1.d

```diff
@@ -74,6 +74,7 @@ targ_size_t paramsize(elem *e, tym_t tyf);
  */
 bool regParamInPreg(Symbol* s)
 {
+    //printf("regPAramInPreg %s\n", s.Sident.ptr);
```

Comment:
I think it should be removed

Reply:
No these debug outputs will come in handy in the future. It's a common practice to do this for DMD.